translations/README_ar_SA.md

```diff
@@ -95,19 +148,21 @@
 
 تُوَزَّع العديد من أنظمة تشغيل ويندوز الأخرى من خلال توفير ISO مُعدّل للنظام. هذا لا ينتهك [شروط خدمة مايكروسوفت](https://www.microsoft.com/en-us/Useterms/Retail/Windows/10/UseTerms_Retail_Windows_10_Arabic.htm) فحسب، بل إنه أيضًا طريقة غير آمنة للتثبيت وقد تُعرِّض جهازك للضرر.
 
-عَقد أطلس شراكة مع فريق Windows Ameliorated لتزويد المستخدمين بطريقة آمنة وقانونية للتثبيت، [AME Wizard](https://ameliorated.io). <br /> باستخدامه، يتوافق أطلس تمامًا مع [شروط خدمة مايكروسوفت](https://www.microsoft.com/en-us/Useterms/Retail/Windows/10/UseTerms_Retail_Windows_10_Arabic.htm).
+عَقد أطلس شراكة مع فريق Windows Ameliorated لتزويد المستخدمين بطريقة آمنة وقانونية للتثبيت، [AME Wizard](https://ameliorated.io). باستخدامه، يتوافق أطلس تمامًا مع [شروط خدمة مايكروسوفت](https://www.microsoft.com/en-us/Useterms/Retail/Windows/10/UseTerms_Retail_Windows_10_Arabic.htm).
 
 ## 🎨 عُدّة البراند
 
-هل ترغب في إنشاء خلفية أطلس الخاصّة بك؟ ربما العبث بشعارنا لعمل التصميم الخاص بك؟ عُدَّتنا في مُتناول الجميع. <br/> [جَرّب العبث بعلامتنا التجاريّة واصنع شيئًا مُذهلًا!](https://cdn.jsdelivr.net/gh/Atlas-OS/Atlas@main/img/brand-kit.zip)
+هل ترغب في إنشاء خلفية أطلس الخاصّة بك؟ ربما العبث بشعارنا لعمل التصميم الخاص بك؟ عُدَّتنا في مُتناول الجميع.
+
+[جَرّب العبث بعلامتنا التجاريّة واصنع شيئًا مُذهلًا!](https://cdn.jsdelivr.net/gh/Atlas-OS/Atlas@main/img/brand-kit.zip)
 
-أيضًا لدينا [مساحة خاصّة للمبدعين](https://forum.atlasos.net/t/art-showcase), حتّى تتمكّن من مُشاركة أفكارك وإبداعاتك مع عباقرة ومبدعين آخرين!
+أيضًا لدينا [مساحة خاصّة للمبدعين](https://forum.atlasos.net/t/art-showcase)، حتّى تتمكّن من مُشاركة أفكارك وإبداعاتك مع عباقرة ومبدعين آخرين!
 
 ## ⚠️ إخلاء مسؤوليّة - Disclaimer
 
 <div align='right'>
 
-https://github.com/Atlas-OS/Atlas#%EF%B8%8F-disclaimer
```

Comment:
keep the old link,

Reply:
it's technically the same link, but as you wish.